### PR TITLE
TST/REF: collect tests by method from generic

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/tests/generic/test_frame.py [hunk 1]
```diff
@@ -76,8 +69,7 @@ def test_get_numeric_data_preserve_dtype(self):
         expected = DataFrame(index=[0, 1, 2], dtype=object)
         self._compare(result, expected)
 
-    def test_metadata_propagation_indiv(self):
-
+    def test_metadata_propagation_indiv_groupby(self):
```
Comment (jreback) [inline pandas/tests/generic/test_frame.py:72]: hmm thought we had a specific module for metadata tests

File: pandas/tests/generic/test_generic.py [hunk 2]
```diff
@@ -534,7 +511,7 @@ def test_pct_change(self, periods, fill_method, limit, exp):
 class TestNDFrame:
     # tests that don't fit elsewhere
 
-    def test_sample(sel):
```
Comment (jreback) [inline pandas/tests/generic/test_generic.py:537]: wow, wonder how this worked

File: pandas/tests/generic/test_frame.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/tests/generic/test_frame.py:72]: @TomAugspurger is there a better home for these?

Comment (TomAugspurger) [reply pandas/tests/generic/test_frame.py:72]: Sorry for the delay. We have some in `pandas/tests/generic/test_finalize.py`.

**Pull request merged**